client/src/app/contact/contactTmpl.html

```diff
@@ -1,7 +1,7 @@
 <md-content id="template-contact" ng-controller="contactCtrl">
     <form name="formContact" ng-submit="send()" class="text-center" novalidate>
         <p><strong>Questions or comments? Leave us a message!</strong></p>
-        <fieldset ng-disabled="submitted">
+        <fieldset ng-disabled="submitting || submitted">
```

Comment:
technically yeah, but once `submitted` is true, the the form is disabled and it can't be in a `submitting` state again